src/data/calendar.ts

```diff
@@ -137,8 +137,12 @@ const getCalendarDate = (dateObj: any): string | undefined => {
   return undefined;
 };
 
-export const getCalendars = (hass: HomeAssistant): Calendar[] =>
-  Object.keys(hass.states)
+export const getCalendars = (
+  hass: HomeAssistant,
+  element?: Element
```

Comment:
```suggestion
  element: Element
```
Can we make it required? We always have element to pass.